code/game/objects/items/robot/robot_parts.dm

```diff
@@ -36,6 +39,40 @@
 	icon_state = LIMB_RIGHT_LEG
 	part = list(LIMB_RIGHT_LEG,LIMB_RIGHT_FOOT)
 
+/obj/item/robot_parts/tail
```

Comment:
i don't like this

Reply:
@ShiftyRail elaborate

Reply:
probably the sheer idea of making and having robot tails in the first place
these tails really should be fully functional organs with behaviours and mechanics (balance aid for tail races, balance debuff for tailless races, being able to fracture it, bleeding, etc) rather than just cosmetic

Reply:
@Eneocho the point of robotails is so if you get a disease that robotizes your limbs, your tail isnt left out

making tails a proper fully damageable organ is for another pr, that requires a huge refactor of damage/organ/hud/everything code

Reply:
fair enoguh